src/SignalR/server/SignalR/test/Microsoft.AspNetCore.SignalR.Tests/HubConnectionHandlerTests.Activity.cs

```diff
@@ -375,7 +375,9 @@ bool ExpectedErrors(WriteContext writeContext)
                 var connectionHandlerTask = await client.ConnectAsync(connectionHandler).DefaultTimeout();
                 client.Dispose();
 
+#pragma warning disable xUnit1030 // Do not call ConfigureAwait(false) in test method
                 await connectionHandlerTask.ConfigureAwait(ConfigureAwaitOptions.SuppressThrowing);
```

Comment:
xunit needs to improve the analyzer to not complain about this.

Reply:
Looks like warning for most uses of `ConfigureAwait` was done on purpose: https://xunit.net/xunit.analyzers/rules/xUnit1030

I think we should just remove this warning in our editor config. We already commonly get off the xunit threads because they cause issues
https://github.com/dotnet/aspnetcore/blob/42af9fe6ddd7c3f9cde04ac003bf97509881873b/src/SignalR/xunit.runner.json#L4